src/java.com/homework6/com/flower/kyiv/Flower.java

```diff
@@ -0,0 +1,12 @@
+package homework6.com.flower.kyiv;
+
+public abstract class Flower {
+    private int price;
+    public Flower(int price) {
+        this.price = price;
+    }
+
+    public String name() {
```

Comment:
Since the flower is an abstract entity, it is more logical to make this method abstract here.